lib/parser/resolve-packages.js

```diff
@@ -0,0 +1,36 @@
+const transform = require('./transform'); //eslint-disable-line
+const {spawn} = require('child_process');
+const path = require('path');
+
+function processPromise(child) {
+	return new Promise(function(resolve, reject) { //eslint-disable-line
+		child.addListener('error', reject);
+		child.addListener('exit', resolve);
+	});
+}
+function spawnChild(pkg) {
+	// spawn('npm', ['install', '--save', pkg], { stdio: 'inherit', customFds: [0, 1, 2] });
+	return spawn('npm', ['install', '--save', pkg]);
+}
+
+module.exports = function resolvePackages(option) {
+	option.packages.filter( (pkg) => {
+		processPromise(spawnChild(pkg)).then( (result) => {
```

Comment:
We handle the error on ourselves and packageModule should be passed to `transform` later. Result is never used since we know it has succeeded if it doesn't make an error